static/js/breakpoint-handler.js

```diff
@@ -22,18 +24,18 @@ function getViewportDimensions() {
  *  enter: callback when breakpoint region entered,
  *  leave: callback when breakpoint region exited
  * }
- * 
+ *
  */
 
-BreakpointHandler = function (opts) {
-    opts || (opts = {});
+function BreakpointHandler(opts) {
```

Comment:
Yes, there's the [func-style](http://eslint.org/docs/rules/func-style.html) rule. I erred on the side of having a fairly strict set of rules because then we can dial back if needed later on instead of realizing we want something enforced and having to refactor a bunch. However, I set much of the stylistic ones to warnings (func-style settings [here](https://github.com/cfpb/cfgov-refresh/pull/329/files#diff-46c54ebc1568fad8993e8986b4a96476R425)), if we were sure we didn't want them at that level even, they can be turned off with a 0.